helm/aws-load-balancer-controller/templates/rbac.yaml

```diff
@@ -57,8 +57,21 @@ rules:
   resources: [services, ingresses]
   verbs: [get, list, patch, update, watch]
 - apiGroups: [""]
-  resources: [nodes, secrets, namespaces, endpoints]
+  resources: [nodes, namespaces, endpoints]
   verbs: [get, list, watch]
+{{- if .Values.clusterSecretsPermissions.allowAllSecrets }}
+- apiGroups: [""]
+  resources: [secrets]
+  verbs: [get, list, watch]
+{{- else if .Values.clusterSecretsPermissions.secretNames }}
+- apiGroups: [""]
+  resources: [secrets]
+{{- with .Values.clusterSecretsPermissions.secretNames }}
+  resourceNames:
```

Comment:
will resourceNames supports namespace as well?

Reply:
This is for convenience. We should require users to create a namespaced role and role binding for better security.

Reply:
removed the secretNames option